topic_test.go

```diff
@@ -920,3 +921,101 @@ func TestWithTopicMsgIdFunction(t *testing.T) {
 		t.Fatal("msg ids are equal")
 	}
 }
+
+func TestTopic_PublishWithKeyInvalidParameters(t *testing.T) {
```

Comment:
can we remove the underscore?

Reply:
sure

Reply:
this is the Elrond pattern used in tests :)

Reply:
done